R/search.R

```diff
@@ -12,9 +12,6 @@ Search_ <- function(.obj = "", body = list(), params = list(), ...){
   } else {
     .obj
   }
-  if (as.character(body) == "[]") {
```

Comment:
@sckott it's unclear to me what this bit tries to achieve. Try `Search_("shakespeare")` to see the error it was throwing.